### ISSUE TITLE
Get wavelength ranges for an observation

### ISSUE THREAD

**Issue by ebuchlin** [solo-spice/sospice#32]
From an observation (Catalog row / FileMetadata object), get a list of wavelength ranges, as interpreted from the `WAVECOV` column; example:
```
WAVECOV = '70.039865-70.839504, 71.61964-72.107225, 74.779191-75.266776, 76.66&'
CONTINUE  '1269-77.441405, 78.348313-79.128449, 96.89013-98.04489, 98.756992-9&'
CONTINUE  '9.372864, 102.90451-103.50113&' / [nm] All WAVEMIN-WAVEMAX, after    
CONTINUE  '' / merging adjacent windows
```
would give
```
[[70.039865, 70.839504], [71.61964, 72.107225], [74.779191, 75.266776], [76.661269, 77.441405], [78.348313, 79.128449], [96.89013, 98.04489], [98.756992, 99.372864], [102.90451, 103.50113]]
```
Optionnally, return in the form of [portion](https://pypi.org/project/portion/) objects (for intervals), and test whether some give wavelength or wavelength range is included in the observed wavelength ranges.

Caveat: because of slit tilt, the actual wavelength ranges depend on the position along the slit.

**Comment by ebuchlin** [comment]
Solved by #33

**Comment by ebuchlin** [comment]
Comments:

* Only return a `portion`, not a list of range boundaries, as having both would make the interface more complicated, and a list of range boundaries is not easily interpretable (and if we have the option to return a `portion`, we need the dependency anyway)
* Only works from data release 3.0, as WAVECOV was not filled properly before (then: the function returns an empty interval)
